changelog.rst

```diff
@@ -12,6 +12,8 @@ Changelog
 2.2.0 (WIP)
 ===========
 
+-   [+] Added a define to enable logging
```

Comment:
Please move that to the bottom of the 2.2.0 section. I want the breaking changes to stand out.